_scicomputing/software_R.md

```diff
@@ -84,19 +84,33 @@ To run `R` on a gizmo node, you can follow the same instructions as for `rhino`
 There is a wrapper script that allows you to run RStudio Server (the web-browser-based version of RStudio)
 on the HPC machines (cluster machines whose names start with `gizmo`).
 
-You can run this with any version of `R` that is available on our shared computing systems.
+You can run this with any version of `R` that is available on our shared computing systems (beginning with `fhR/4.1.2-foss-2021b`).
 
 Here are the steps to run this wrapper.
 
 * Grab a node using the [grabnode](/compdemos/first_rhino/#logging-on-to-gizmo-via-grabnode) command. Specify how many CPUs and how much memory you want, and how many days you want to have control of the node. Remember that you can launch `slurm` jobs within RStudio, so you may not need to ask for a lot of computing power for your RStudio machine.
-* Once you are on the node you grabbed, choose a version of `R` by using the [module load](/scicomputing/compute_environments/#how-to-use-environment-modules) command (`ml` for short). (Example: `fhR/4.0.2-foss-2019b`). **NOTE**: At present, `fhR/4.0.2-foss-2019b` is the *only* R module that will work with RStudio Server.
-* Run the `launch_rstudio_server` command. This will produce a URL that you can paste into your browser. (This URL only works inside the Hutch network, so you need to be on campus or using VPN.)
+* Run the `start_rstudio_server` command. This will produce a URL that you can paste into your browser. (This URL only works inside the Hutch network, so you need to be on campus or using VPN.)
```

Comment:
Do we need to change the command name used to start the server?

Reply:
I don't mind the change.  Makes it less likely that you'll accidentally do it the "old way" and run into issues.  But any shift will initially confuse people.  Given how much confusion there is though about Rstudio and R versions though, I feel like this extra change that allows them to never think about R versions again hopefully, is worth it.